### DISCUSSION TITLE
Metro map

### DISCUSSION THREAD

**Discussion by Ato58** [organicmaps/organicmaps#6425] [Ideas]
Excellent maps... Only one request: I would like the Metro to be a permanent part of the map – without graying out the background.
Hello, the Metro map is the most important function for pedestrian navigation in a large foreign city. (not only for public transport navigation). It determines how far I am from metro stops, it determines where a hotel, sightseen or restaurant is located. Metro  map makes orientation much easier. It has only Google maps and old Here we go maps.  Just like the "Terrain" layer is permanently on, the "Metro" layer could be turned on as well. (just map in the background is not gray and indistinct.) Thank you

**Comment by biodranik** [comment]
Please follow/elaborate on #1577
